public/static/docs/api-reference/get_url.md

```diff
@@ -0,0 +1,111 @@
+### `dvc.api.get_url(path, repo=None, rev=None, remote=None)`
+
+Returns the URL to the storage location of a data file or directory tracked by
+DVC.
+
+```py
+import dvc.api
+
+resource_url = dvc.api.get_url(
+    'get-started/data.xml',
+    repo='https://github.com/iterative/dataset-registry')
+
+# resource_url =
+# https://remote.dvc.org/dataset-registry/a3/04af...
+```
+
+All **parameter** types are
+[string](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str).
+
+## Description
+
+Returns the URL string of the physical location (in a
+[DVC remote](/doc/command-reference/remote)) where a target file or directory
+(<abbr>artifact</abbr>), specified by its `path` in a `repo` (<abbr>DVC
+project</abbr>), is stored. The URL is formed by reading the corresponding
+[DVC-file](/doc/user-guide/dvc-file-format) (see [Examples](#examples) below) as
+well as the project's
+[remote configuration](/doc/command-reference/config#remote).
+
+The current project is used by default (the current working directory tree is
```

Comment:
If `repo` URL is not provided it is trying to analyze if the current working directory (`.`) is part the DVC repo and use it as a source. See example ...

Reply:
This has already been moved back to the params section.